pkg/sensors/base/base.go

```diff
@@ -64,6 +68,27 @@ var (
 	sensorInit sync.Once
 )
 
+func setupExitProgram() {
+	failed := true
+	ks, err := ksyms.KernelSymbols()
+	if err == nil {
+		has_acct_process := ks.IsAvailable("acct_process")
+		has_disassociate_ctty := ks.IsAvailable("disassociate_ctty")
+
+		/* Preffer acct_process over disassociate_ctty */
+		if has_acct_process {
+			Exit.Attach = "acct_process"
+			Exit.Label = "kprobe/acct_process"
+		} else if has_disassociate_ctty {
+			Exit.Attach = "disassociate_ctty"
+			Exit.Label = "kprobe/disassociate_ctty"
+		}
+		failed = !(has_acct_process || has_disassociate_ctty)
+	}
+	logger.GetLogger().WithField("failed", failed).
+		Infof("Exit probe on %s", Exit.Attach)
```

Comment:
Is there a way to recover from this? If not, I think we should just add a `.Fatal()` here so we die early.

Reply:
we would start without hooked exit probe.. so it's probably better to bail out then do some strange things